packages-rs/drainpipe/src/db.rs

```diff
@@ -0,0 +1,41 @@
+use diesel::prelude::*;
+use diesel::sqlite::SqliteConnection;
+
+pub fn db_connect(database_url: &String) -> anyhow::Result<SqliteConnection> {
+    SqliteConnection::establish(&database_url).map_err(Into::into)
+}
+
+pub fn update_seq(conn: &mut SqliteConnection, seq: i32) -> anyhow::Result<()> {
+    use crate::schema::drainpipe::dsl::*;
+
+    diesel::insert_into(drainpipe)
+        .values(seq.eq(&seq))
+        .execute(conn)?;
```

Comment:
I get an error here (and in the other method) and I'm not sure why, I'm following the example pretty much exactly

```
error[E0271]: type mismatch resolving `<NoFromClause as AppearsInFromClause<table>>::Count == Once`
    --> packages-rs/drainpipe/src/db.rs:13:18
     |
13   |         .execute(conn)?;
     |          ------- ^^^^ expected `Once`, found `Never`
     |          |
     |          required by a bound introduced by this call
     |
note: required for `schema::drainpipe::columns::seq` to implement `AppearsOnTable<NoFromClause>`
    --> packages-rs/drainpipe/src/schema.rs:14:9
     |
14   |         seq -> Integer,
     |         ^^^
     = note: associated types for the current `impl` cannot be restricted in `where` clauses
     = note: 1 redundant requirement hidden
     = note: required for `&schema::drainpipe::columns::seq` to implement `AppearsOnTable<NoFromClause>`
     = note: required for `ColumnInsertValue<schema::drainpipe::columns::seq, &schema::drainpipe::columns::seq>` to implement `InsertValues<_, schema::drainpipe::table>`
     = note: required for `ValuesClause<ColumnInsertValue<seq, &seq>, table>` to implement `QueryFragment<_>`
     = note: 1 redundant requirement hidden
     = note: required for `InsertStatement<table, ValuesClause<ColumnInsertValue<seq, &seq>, table>>` to implement `QueryFragment<_>`
     = note: required for `InsertStatement<table, ValuesClause<ColumnInsertValue<seq, &seq>, table>>` to implement `ExecuteDsl<_, _>`
note: required by a bound in `diesel::RunQueryDsl::execute`
```